sp800-63-3/cover.md

```diff
@@ -187,77 +188,78 @@ The terms “SHOULD” and “SHOULD NOT” indicate that among several possibil
+Each document has adopted verbs that are internationally recognized in standards organizations as normative and requirements-based. When used in a normative statement in this publication, they are CAPITALIZED for ease of identification.  For example, the use of SHALL is used to denote a mandatory requirement, while the use of SHOULD refers to a technique, technology, or process that is recommended but not mandatory.  For more details on the definitions of these terms see the [Requirements Notation and Conventions](#notation) at the beginning of each document.
 
-Each document has adopted verbs that are internationally recognized in standards organizations as normative and requirements-based. For legibility, they are CAPITALIZED throughout the document for ease of identification.  For example, the use of SHALL is used to denote a mandatory requirement, while the use of SHOULD refers to a technique, technology, or process that is recommended but not mandatory.  For more details on the definitions of these terms see the [Requirements Notation and Conventions](#notation) at the beginning of each document.
+These documents may inform, but does not restrict or constrain, the development or use of standards for application outside of the Federal government, such as e-commerce transactions.
 
 These guidelines are organized as follows:
 
-**SP 800-63-3 Digital Authentication Guideline**
+**SP 800-63-3 Digital Identity Guidelines** (This document)
 
-This document provides an overview of general authentication frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _This document is informative._
+SP 800-63-3 provides an overview of general identity frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _This document contains only informative material._
 
 [**SP 800-63A Enrollment and Identity Proofing**](#800-63a)
 
-NIST SP 800-63-A deals with how an individual can prove their identity and become enrolled as a valid user within an identity system. It provides guidelines on processes by which an individual can enrolled and identity proofed in both remote and in-person scenarios. _This document contains both normative and informative material._
+NIST SP 800-63-A addresses how applicants can prove their identities and become enrolled as valid subjects within an identity system. It provides guidelines for processes by which applicants can both proof and enroll at one of three different levels of risk mitigation in both remote and physically-present scenarios. _This document contains both normative and informative material._
```

Comment:
fixed - i was trying not to get too in the weeds on the valid terms for each volume, but i see the point.